src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java

```diff
@@ -293,6 +293,44 @@ public void testGetUser_successful() throws Exception {
         Mockito.verify(mockApiClient).get("https://api.sandbox.hyperwallet.com/rest/v4/users/test-user-token", userResponse.getClass());
     }
 
+    @Test
+    public void getUser_WithVerificationDocumentAndRejectReasons() throws Exception {
+        Hyperwallet client = new Hyperwallet("test-username", "test-password");
+        HyperwalletApiClient mockApiClient = createAndInjectHyperwalletApiClientMock(client);
+        HyperwalletUser hyperwalletUser = getHyperwalletUser();
+        final String url = "https://api.sandbox.hyperwallet.com/rest/v4/users/" + "test-userToken";
+        Mockito.when(mockApiClient.get(Mockito.eq(url), Mockito.eq(HyperwalletUser.class))).thenReturn(hyperwalletUser);
+
+        HyperwalletUser user = client.getUser("test-userToken");
+
+        Mockito.verify(mockApiClient).get(Mockito.eq(url), Mockito.eq(HyperwalletUser.class));
+        assertThat(user.getToken(), is(hyperwalletUser.getToken()));
```

Comment:
minor: can assertAll be used here?

Reply:
java-sdk module doesn't have a junit-jupiter as a maven dependency, it doesn't allow me to use assertAll.